File: classes/parte-9-ingenieria-de-software-poliglota/143-dependencias-versiones-y-lockfiles/implementaciones/c/main.c
#include <stdio.h>

int main(void) {
    long ma, me, pa;
    if (scanf("%ld.%ld.%ld", &ma, &me, &pa) != 3) return 1;
    printf("mayor=%ld menor=%ld parche=%ld\n", ma, me, pa);
    return 0;
}

File: classes/parte-9-ingenieria-de-software-poliglota/143-dependencias-versiones-y-lockfiles/implementaciones/rust/main.rs
use std::io::Read;

fn main() {
    let mut s = String::new();
    std::io::stdin().read_to_string(&mut s).unwrap();
    let v: Vec<i64> = s.trim().split('.').map(|x| x.parse().unwrap()).collect();
    println!("mayor={} menor={} parche={}", v[0], v[1], v[2]);
}
